test/only.js

```diff
@@ -0,0 +1,13 @@
+QUnit.module( "QUnit.only" );
+
+QUnit.test( "implicitly skipped test", function( assert ) {
+	assert.ok( false, "test should be skipped" );
+});
+
+QUnit.only( "only run this test", function( assert ) {
+	assert.ok( true, "only this test should run" );
+});
+
+QUnit.test( "another implicitly skipped test", function( assert ) {
+	assert.ok( false, "test should be skipped" );
+});
```

Comment:
Sounds good to me. I'll try and get something up with this feedback tonight or tomorrow. Thanks for the feedback.